py_june/urls.py

```diff
@@ -23,4 +23,5 @@
     path('admin/', admin.site.urls),
     path('', include('core.urls')),
     path('forum/', include('categories.urls')),
+    path('', include('users.urls')),
```

Comment:
better to use prefix for every module and use namespace for clear url management
```suggestion
    path('users', include('users.urls', namespace='users')),
```